Lib/copy.py

```diff
@@ -177,21 +179,13 @@ def deepcopy(x, memo=None, _nil=[]):
         _keep_alive(x, memo) # Make sure x lives at least as long as d
     return y
 
+from _copy import deepcopy
```

Comment:
The stdlib is reused by other Python implementations like PyPy. Please make the ``_copy`` accelerator module option and have it fall back to a pure Python impementation.